src/ParametersHandler/YarpImplementation/include/BipedalLocomotion/ParametersHandler/YarpImplementation.tpp

```diff
@@ -21,56 +21,66 @@ namespace ParametersHandler
 template <typename T>
 bool YarpImplementation::getParameterPrivate(const std::string& parameterName, T& parameter) const
 {
-    if (m_lists.find(parameterName) != m_lists.end()) // A list is called with the same name of the parameter we are searching
-    {
-        return m_lists.at(parameterName)->getParameter(parameterName, parameter);
-    }
+    if constexpr (std::is_scalar<T>::value || is_string<T>::value)
+        // Scalars and strings are retrieved using getElementFromSearchable() function
+        return YarpUtilities::getElementFromSearchable(m_container, parameterName, parameter);
```

Comment:
I actually do not understand this modification. Removing this check
```cpp
if (m_lists.find(parameterName) != m_lists.end())
```
we risk breaking something, no?

Reply:
Mmh I have to check more in detail, but I think that the `get*FromSearchable` already return false if the parameter name (key) is not found, right?

Reply:
As per https://github.com/dic-iit/bipedal-locomotion-framework/pull/134#issuecomment-725388389, the `IParametersHandler` modification have been removed from this PR.